components/script/dom/event.rs

```diff
@@ -330,8 +330,8 @@ impl Runnable for EventRunnable {
 
     fn handler(self: Box<EventRunnable>) {
         let target = self.target.root();
-        let bubbles = self.bubbles.clone();
-        let cancelable = self.cancelable.clone();
+        let bubbles = self.bubbles;
+        let cancelable = self.cancelable;
```

Comment:
Nit: pass `self.bubbles` and `self.cancelable` directly instead of binding them to local variables.